resource-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/resource/BasicPersistenceTest.java

```diff
@@ -75,7 +75,7 @@
      * F   F  F
      */
 
-    private static final SimpleConnectionDetail DETAIL = new SimpleConnectionDetail().uri("test-data-service");
+    private static final SimpleConnectionDetail DETAIL = new SimpleConnectionDetail().serviceName("test-data-service");
```

Comment:
```suggestion
    private static final SimpleConnectionDetail DETAIL = new SimpleConnectionDetail().serviceName("data-service-mock");
```
Consistent naming convention